src/component/update.modal.tsx

```diff
@@ -0,0 +1,133 @@
+"use client";
+import { useState, useEffect } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import Modal from "react-bootstrap/Modal";
+import { toast } from "react-toastify";
+import { mutate } from "swr";
+
+interface IProps {
+  showModalUpdate: boolean;
+  setShowModalUpdate: (v: boolean) => void;
+  blog: IBlog | null;
+  setBlog: (value: IBlog | null) => void;
+}
+
+function UpdateModal(props: IProps) {
+  const { showModalUpdate, setShowModalUpdate, blog, setBlog } = props;
+
+  const [id, setId] = useState<number>(0);
+  const [title, setTitle] = useState<string>("");
+  const [author, setAuthor] = useState<string>("");
+  const [content, setContent] = useState<string>("");
+
+  useEffect(() => {
+    if (blog && blog.id) {
+      setId(blog.id);
+      setTitle(blog.title);
+      setAuthor(blog.author);
+      setContent(blog.content);
+    }
+  }, [blog]);
+  const handleSubmit = () => {
+    if (!title) {
+      toast.error("Not empty title !");
+      return;
+    }
+    if (!author) {
+      toast.error("Not empty author !");
+      return;
+    }
+    if (!content && !content && !author) {
+      toast.error("Not empty data !");
+      return;
+    }
+
+    if (!content) {
+      toast.error("Not empty content !");
+      return;
+    }
+
+    fetch(`https://652c0736d0d1df5273ef0e4e.mockapi.io/api/v1/blogs/${id}`, {
+      method: "PUT",
+      headers: {
+        Accept: "application/json, text/plain, */*",
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({ title, author, content }),
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        if (res) {
+          toast.success("Update succeed");
+          handleCloseModal();
+          mutate("https://652c0736d0d1df5273ef0e4e.mockapi.io/api/v1/blogs"); // gọi lại data để show ra mà không cân reload
+        }
+      });
+  };
+
+  const handleCloseModal = () => {
+    setAuthor("");
+    setContent("");
+    setTitle("");
+    setBlog(null); //khi mình nhấn edit nhiều nó sẽ không trống dữ liệu
+    setShowModalUpdate(false);
+  };
+
+  return (
+    <>
+      <Modal
+        show={showModalUpdate}
+        onHide={() => handleCloseModal()}
+        backdrop="static"
+        keyboard={false}
+        size="lg" // modal dài ra
+      >
+        <Modal.Header closeButton>
+          <Modal.Title>Update</Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+          <Form>
+            <Form.Group className="mb-3">
+              <Form.Label>Enter the title</Form.Label>
+              <Form.Control
+                type="text"
+                placeholder="Enter the title"
+                value={title}
+                onChange={(e) => setTitle(e.target.value)}
+              />
+            </Form.Group>
+            <Form.Group className="mb-3">
+              <Form.Label>Enter the Author</Form.Label>
```

Comment:
@HoVanDi - No, I mean that you should update the "placeholder" value to "Enter the author"
And keep the Label value as before: "Author"

Reply:
Thank you for your feedback and I have updated as you commented